README.md

```diff
@@ -8,9 +8,9 @@ This role depends on (postgres)[https://github.com/stuvusIT/postgresql], (redis)
 
 ## Role Variables
 
-To enable a variable config for gitlab and since GitLab is also using yaml files you can pretty much write your GitLab config into this role, 
+To enable a variable config for gitlab and since GitLab is also using yaml files you can pretty much write your GitLab config into this role,
 with the exception that we merge the given config with the default options of GitLab.
```

Comment:
why the newline in the middle of this sentence?

Reply:
because when you view this in vim you need to scrol lover

Reply:
Bad choice for a markdown viewer, I would say. See https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line, "The result can be spectacular."